src/govuk/components/header/_index.scss

```diff
@@ -43,6 +43,10 @@
     // High Contrast Mode
     forced-color-adjust: none;
 
+    @media (forced-colors: active) {
+      color: linktext;
+    }
+
     // But remove it if there's nothing after the logo to keep hover and focus
     // states neat
```

Comment:
I know it wasn't changed in this PR, but this comment is kind of confusing now that there is stuff between setting and removing `margin-right`

Reply:
Great catch. I think there's a couple of options…

We could move the color adjust properties to after the pseudo-selector. However, I think we generally try and avoid having nested selectors in the middle of a ruleset.

```scss
  .govuk-header__logotype {
    display: inline-block;

    // Add a gap between logo and any product name
    margin-right: govuk-spacing(1);

    // But remove it if there's nothing after the logo to keep hover and focus
    // states neat
    &:last-child {
      margin-right: 0;
    }

    // Prevent readability backplate from obscuring underline in Windows
    // High Contrast Mode
    forced-color-adjust: none;

    @media (forced-colors: active) {
      color: linktext;
    }
  }
```

We could also just update the comment to make it less confusing, but the two related declarations are still a little divorced from each other:

```scss
  .govuk-header__logotype {
    display: inline-block;

    // Add a gap after the logo in case it's followed by a product name. This
    // gets removed later if the logotype is a :last-child.
    margin-right: govuk-spacing(1);

    // Prevent readability backplate from obscuring underline in Windows
    // High Contrast Mode
    forced-color-adjust: none;

    @media (forced-colors: active) {
      color: linktext;
    }

    // Remove the gap after the logo if there's no product name to keep hover
    // and focus states neat
    &:last-child {
      margin-right: 0;
    }
  }
```

Or… something else I haven't thought of.

Thoughts?

Reply:
Second option looks good to me.

Reply:
Is the rule about nested declarations purely a stylistic thing, or does the location affect the generated CSS?